scoper.php

```diff
@@ -122,42 +122,36 @@ function (string $filePath, string $prefix, string $content): string {
         },
 
         // unprefixed ContainerConfigurator
-        function (string $filePath, string $prefix, string $content): string {
-            // keep vendor prefixed the prefixed file loading; not part of public API
-            // except @see https://github.com/symfony/symfony/commit/460b46f7302ec7319b8334a43809523363bfef39#diff-1cd56b329433fc34d950d6eeab9600752aa84a76cbe0693d3fab57fed0f547d3R110
-            if (str_contains($filePath, 'vendor/symfony') && ! str_ends_with(
-                $filePath,
-                'vendor/symfony/dependency-injection/Loader/PhpFileLoader.php'
-            )) {
-                return $content;
-            }
-
-            return Strings::replace(
-                $content,
-                '#' . $prefix . '\\\\Symfony\\\\Component\\\\DependencyInjection\\\\Loader\\\\Configurator\\\\ContainerConfigurator#',
-                'Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator'
-            );
-        },
+        fn (string $filePath, string $prefix, string $content): string => // keep vendor prefixed the prefixed file loading; not part of public API
+// except @see https://github.com/symfony/symfony/commit/460b46f7302ec7319b8334a43809523363bfef39#diff-1cd56b329433fc34d950d6eeab9600752aa84a76cbe0693d3fab57fed0f547d3R110
+str_contains($filePath, 'vendor/symfony') && ! str_ends_with(
+    $filePath,
+    'vendor/symfony/dependency-injection/Loader/PhpFileLoader.php'
+) ? $content : Strings::replace(
+            $content,
+            '#' . $prefix . '\\\\Symfony\\\\Component\\\\DependencyInjection\\\\Loader\\\\Configurator\\\\ContainerConfigurator#',
+            'Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator'
+        ),
 
         // get version for prefixed version
-        function (string $filePath, string $prefix, string $content): string {
-            if (! \str_ends_with($filePath, 'src/Configuration/Configuration.php')) {
-                return $content;
-            }
-
-            // @see https://regex101.com/r/gLefQk/1
-            return Strings::replace($content, '#\(\'rector\/rector-src\'\)#', "('rector/rector')");
-        },
+        fn (string $filePath, string $prefix, string $content): string => \str_ends_with(
+            $filePath,
+            'src/Configuration/Configuration.php'
+        ) ? Strings::replace(
+            $content,
+            '#\(\'rector\/rector-src\'\)#',
+            "('rector/rector')"
+        ) : $content,
 
```

Comment:
This is also extremely hard to read. Maybe it would be better not to extend this rule and keep it as it is.
Harder readability code is not our aim.

Reply:
ok 👍

Reply:
I am closing it then ;)